app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes
+
+  def three_recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+end
```

Comment:
- You have done great so far :+1: But there are a few requirements you need to meet before moving to the next milestone. :+1: :100: 

- - Requirement Screenshot:

![Selection_054](https://github.com/EvansSnave/Blog_app_rails/assets/107632780/6c9925f5-415a-4c7c-ab37-43bf9ecccc30)

- [ ] According to the project requirement, you need to add validations to the user model to make sure that a name is always present t when a user is created and also that PostsCounter must be an integer greater than or equal to zero. Currently, you haven't implemented this. I suggest you do as it is part of the requirement, :+1: :+1: :100: Here is a snippet of what the first validation could look like:
```ruby
validates :name, presence: true
```